packages/migrations/src/actions/2024.11.19T00.00.00.create-preflight-scripts.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'node:path';
+import { type MigrationExecutor } from '../pg-migrator';
+
+const filename = path.parse(import.meta.url).base;
+
```

Comment:
TODO @dimaMachina Since we bundle this file, this migration will be ran as `index.js` instead of this file name.

Please hardcode the filename here. as a string

Reply:
<img width="471" alt="image" src="https://github.com/user-attachments/assets/59419893-3ca7-49f3-a9a1-2b65b4b16591">

Reply:
hardcoded